### PR TITLE
Release/dev

### PR DESCRIPTION
**Pull request by bablubharath**

### CODE REVIEW

**Comment by bablubharath** [comment]
files ok

**Comment by bablubharath** [comment]
ok

File: variables.tf [hunk 1]
```diff
@@ -1,12 +1,12 @@
 variable "recovery_window_in_days" {
   description = "Specifies the number of days that AWS Secrets Manager waits before it can delete the secret. This value can be 0 to force deletion without recovery or range from 7 to 30 days."
   type        = number
-  default     = 30
+  default     = 7
```
Comment (lgallard) [inline variables.tf:4]: @bablubharath it makes sense. Please remove the Jenkins file to merge this PR!

File: Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,19 @@
+pipeline {
```
Comment (lgallard) [inline Jenkinsfile:1]: @bablubharath why a Jenkins file in here? Can you remove it please?

**Review by lgallard — CHANGES_REQUESTED** [review]

File: providers.tf [hunk 3]
```diff
@@ -0,0 +1,3 @@
+provider "aws" {
+  region = "us-east-1"
+}
```
Comment (stevie-) [inline providers.tf:3]: Providers should not be present in modules. If aws provider is required one uses [required_providers](https://github.com/lgallard/terraform-aws-secrets-manager/blob/master/versions.tf#L4) setting

Delete this file.

**Review by stevie- — CHANGES_REQUESTED** [review]

File: providers.tf (hunk 3 above)
Comment (lgallard) [reply providers.tf:3]: Agree!

**Comment by lgallard** [comment]
@bablubharath closing this PR. In case you want to include changes please consider the suggestions in this thread! Thanks!

**Pull request closed without merging**